other/workshop-app-styles.css

```diff
@@ -16,3 +16,13 @@ hr {
   margin-top: 30px;
   margin-bottom: 30px;
 }
+
+/* 
+This hides the Create-React-App ErrorOverlay that will overlay the whole application when React fails to compile. The problem here is that it also overlays the workshop instructions with the example
+
+Reference: https://github.com/kentcdodds/react-workshop-app/issues/4
+
+*/
+iframe {
```

Comment:
Let's leave the isolated pages out of this. On the isolated pages the error overlay can be kinda handy.

```suggestion
body:not([class*="isolated"]) iframe {
```

Reply:
oh good call